Lucid/Core/RelationshipController.swift

```diff
@@ -443,47 +405,29 @@ public extension RelationshipController {
         private var mainContext: ReadContext
 
         private let rootEntities: (
-            once: Signal<AnySequence<Graph.AnyEntity>, ManagerError>,
-            continuous: SafeSignal<AnySequence<Graph.AnyEntity>>
+            once: AnyPublisher<AnySequence<Graph.AnyEntity>, ManagerError>,
+            continuous: AnySafePublisher<AnySequence<Graph.AnyEntity>>
         )
 
         private let relationshipManager: RelationshipManager?
 
-        private init(_ rootEntities: (once: Signal<QueryResult<E>, ManagerError>, continuous: SafeSignal<QueryResult<E>>),
-                     in mainContext: ReadContext,
-                     relationshipManager: RelationshipManager?) {
+        public init(rootEntities: (once: AnyPublisher<QueryResult<E>, ManagerError>, continuous: AnySafePublisher<QueryResult<E>>),
+                                in mainContext: ReadContext,
```

Comment:
This indentation looks wrong but I guess it was Xcode's doing

Reply:
Ah, yeah, there might be a few of those.

For Monday Rod, Monday!